src/api/parser/src/utils/indexer.js

```diff
@@ -0,0 +1,173 @@
+const { setIntervalAsync, clearIntervalAsync } = require('set-interval-async/dynamic');
+
+const { ELASTIC_MAX_RESULTS_PER_PAGE = 5, POSTS_URL = 'http://localhost/v1/posts' } = process.env;
+const { logger } = require('@senecacdot/satellite');
+const { client } = require('../lib/elastic');
```

Comment:
Should we use `elasticsearch client` from satellite?

Reply:
Yes, since we're using satellite's `redis` client, I think it'd make sense to do the same with `elasticsearch`.